channels/pl.m3u

```diff
@@ -1,53 +1,79 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="https://i.imgur.com/c59VWgF.jpg" group-title="Music",4FUN TV
 https://stream.4fun.tv:8888/hls/4f_high/index.m3u8
+#EXTINF:-1 tvg-id="WTK PL" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",dlaCiebieTV
+http://94.246.128.53:1935/tv/_definst_/dlaCiebieTv/playlist.m3u8
+#EXTINF:-1 tvg-id="WTK PL" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",Echo24
+https://insysgo.live.e57-po.insyscd.net/echo24.720.smil/chunklist_b4696000_slpl.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/hJAoLIq.png" group-title="",EZO TV
 http://live.ezotv.pl:1935/live/EZOTV/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="",Sfera TV
+http://stream.sferatv.pl:1935/sferalive/smil:sferalive.smil/chunklist_w1591103044_b1128000_slpl.m3u8
```

Comment:
If possible to create an adaptive link for this and echo24, would be preferred, but this is okay.

Reply:
Yes, I will fix that by using better one grabbed from website: 
http://stream.sferatv.pl:1935/sferalive/smil:sferalive.smil/playlist.m3u8

Reply:
Can you help? it don't work somehow ;/ It work with chunk one. I have fealing that chunk file won't work in future